browser/translate/buildflags/buildflags.gni

```diff
@@ -1,6 +1,11 @@
 import("//brave/build/config.gni")
 import("//build/config/features.gni")
+import("//extensions/buildflags/buildflags.gni")
 
 declare_args() {
-  enable_brave_translate = !is_release_channel
+  enable_brave_translate_go = false
+}
```

Comment:
nit - you only need one declare_args section here

Reply:
I'm actually getting below build error when doing `enable_brave_translate_extension = !enable_brave_translate_go && enable_extensions` if using one declare_args.
`ERROR at //brave/browser/translate/buildflags/buildflags.gni:7:39: Reading a variable defined in the same declare_args() call.`